kong/clustering/rpc/manager.lua

```diff
@@ -172,8 +173,17 @@ function _M:call(node_id, method, ...)
 
   local params = {...}
 
+  ngx_log(ngx_DEBUG,
```

Comment:
Should we use `ngx.INFO` or `ngx.NOTICE`, it may be easier to log without turning log level to debug.